zerver/lib/utils.py

```diff
@@ -33,3 +33,10 @@ def process_list_in_batches(
             break
         process_batch(items)
         offset += chunk_size
+
+
+def optional_bytes_to_mib(value: Optional[int]) -> Optional[int]:
+    if value is None:
+        return None
+    else:
+        return value >> 20
```

Comment:
This is how we convert bytes to GiB (`>> 30`), so I'm following the pattern here. I guess this wouldn't be correct if a float had to be returned, but I think we can assume this can always be an integer here as well?

Reply:
Yeah that seems fine.